mpc-spec/src/primitives/auth_share.rs

```diff
@@ -0,0 +1,13 @@
+//! This module defines the interface for share authentication.
+use super::mac::{Mac, MacKey};
+
+/// An authenticated share of a bit.
+#[allow(dead_code)] // TODO: Remove this later.
+pub struct AuthShare {
+    /// Party i's share of the bit
+    share: bool,
+    /// MACs on the shared bit provided by the other parties
+    macs: Vec<(usize, Mac)>,
+    /// Keys for authenticating the other parties' shares of the bit
+    keys: Vec<(usize, MacKey)>,
+}
```

Comment:
Good point!
This is in part related to the as yet missing functionality for creating an authenticated share. I'm not fully sure of the API here yet, so I did not put in a dummy function signature.
I will make the fields public on the crate level, since it keeps the spec more concise.